### PR TITLE
[BPK-1618] Add copywriting page to docs

### PR DESCRIPTION
**Pull request by shaundon**
**To do**

* [x] Add new routes
* [x] Skeleton page
* [x] Get it in the sidebar (commented out)
* [x] Add to home page (commented out)

### CODE REVIEW

File: packages/bpk-docs/src/layouts/links.js [hunk 1]
```diff
@@ -608,4 +608,15 @@ export default [
       },
     ],
   },
+  {
+    id: 'STYLE_GUIDE',
+    category: 'Style guide',
+    sort: true,
+    hero: {
+      url: `/${DesignTokensImage}`, // TODO
```
Comment (shaundon) [inline packages/bpk-docs/src/layouts/links.js]: Will be fixed in the subsequent card to flesh out the page and hero image component.

File: packages/bpk-docs/src/pages/CopywritingPage/CopywritingPage.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+
+const blurb = [<IntroBlurb>Copywriting page!</IntroBlurb>];
+
+const CopywritingPage = () => (
+  <DocsPageBuilder title="Copywriting" blurb={blurb} />
+);
+
+export default CopywritingPage;
```
Comment (shaundon) [inline packages/bpk-docs/src/pages/CopywritingPage/CopywritingPage.js]: To be fleshed out in another card.

File: packages/bpk-docs/src/components/neo/HomePageCard/HomePageCard.js [hunk 3]
```diff
@@ -47,28 +48,30 @@ const HomePageCard = (props: Props) => {
...
 
-  if (className) {
-    classNames.push(className);
-  }
-  const iconClassNames = [getClassName('bpkdocs-home-page-card__icon')];
+  const classNames = getClassName(
+    'bpkdocs-home-page-card',
+    fullWidth && 'bpkdocs-home-page-card--full-width',
+    className,
+  );
```
Comment (matthewdavidson) [inline packages/bpk-docs/src/components/neo/HomePageCard/HomePageCard.js]: nice 😎

File: packages/bpk-docs/src/layouts/links.js (hunk 1 above)
Comment (matthewdavidson) [inline packages/bpk-docs/src/layouts/links.js]: It looks like you've got the image in this PR? (packages/bpk-docs/src/static/style_guide.jpg)

Comment (shaundon) [reply packages/bpk-docs/src/layouts/links.js]: That's different - the hero image and card images aren't exactly the same. E.g. we have `components.jpg` and `components_hero.jpg` too.

**Pull request merged**